src/main/resources/xl-rules.xml

```diff
@@ -131,5 +130,20 @@
         </steps>
     </rule>
 
+	<rule name="datical.DeployedProject.StatusDetail.After.CREATE" scope="deployed">
+        <conditions>
+            <type>datical.DeployedProject</type>
+            <operation>CREATE</operation>
+            <operation>MODIFY</operation>
+        </conditions>
+        <steps>
+            <os-script>
```

Comment:
Can we do this using a Jython step (maybe using requests). That would remove the dependency on `curl`.